app/src/components/layout/LogoComponents/ConfigurableLogo.tsx

```diff
@@ -5,32 +5,39 @@ import { loadConfig } from 'src/utils';
 import { Link } from 'react-router-dom';
 import { pxToRem } from 'src/styled-theme';
 import { useAppWidth } from 'src/hooks';
+import { getIsFirstVisit, useAppSelector } from 'src/store';
 
 export const ConfigurableLogo: React.FC = () => {
-  const { logoUrl } = loadConfig();
+  const { logoUrl, logoSize } = loadConfig();
   const { isMobile } = useAppWidth();
+  const isFirstVisit = useAppSelector(getIsFirstVisit);
 
   return (
-    <Link to="/">
-      <ConfigurableLogoLink isMobile={isMobile}>
+    <ConfigurableLogoContainer
+      logoSize={logoSize}
+      isMobile={isMobile}
+      isFirstVisit={isFirstVisit}>
+      <Link to="/">
         <ConfigLogo src={logoUrl} alt="configuration logo" />
-      </ConfigurableLogoLink>
-    </Link>
+      </Link>
+    </ConfigurableLogoContainer>
   );
 };
 
-export const ConfigurableLogoLink = styled.div<{ isMobile: boolean }>`
-  display: flex;
-  justify-content: start;
-  align-items: center;
-  text-decoration-line: none;
-  contain: content;
-  min-width: ${pxToRem(200)};
-  max-height: ${({ isMobile }) =>
-    isMobile ? `${pxToRem(55)}` : `${pxToRem(45)}`};
-  max-width: ${({ isMobile }) =>
-    isMobile ? `${pxToRem(250)}` : `${pxToRem(200)}`};
-  padding-top: ${({ isMobile }) => (isMobile ? `${pxToRem(7)}` : '0')};
+export const ConfigurableLogoContainer = styled.div<{
+  isMobile: boolean;
+  isFirstVisit: boolean;
+  logoSize: number;
+}>`
+  width: ${({ logoSize }) => logoSize}%;
+  min-width: ${({ isFirstVisit, isMobile }) =>
+    isFirstVisit || isMobile ? `${pxToRem(0)}` : `${pxToRem(0)}`};
+  max-width: ${({ isFirstVisit, isMobile }) =>
+    isFirstVisit || isMobile ? `${pxToRem(345)}` : `${pxToRem(500)}`};
+  padding: ${({ isMobile }) =>
+    isMobile
+      ? `${pxToRem(20)} 0 ${pxToRem(20)} ${pxToRem(10)}`
+      : `${pxToRem(15)} 0`};
```

Comment:
Min width here is the same in both scenarios.  Can we also use css media queries to determine if we are in a mobile view?

Reply:
Updated!